File: 02- Fundamentals/04- Exercise- Functions/04. Odd and Even Sum.py
def check(a):
    sum_even = 0
    sum_odd = 0
    for digit in a:
        if int(digit) % 2 == 0:
            sum_even += int(digit)
        else:
            sum_odd += int(digit)

    print(f"Odd sum = {sum_odd}, Even sum = {sum_even}")

number_input = input()
check(number_input)

File: 08-JS/02-functions-and-statements/04-odd-and-even-sum.js
function oddAndEvenSum(numberInput) {
 
    numberInputAsString = String(numberInput);
    arr = [...numberInputAsString];
 
    let sumEven = 0;
    let sumOdd = 0;
 
    output = '';

    for (let index = 0; index < arr.length; index++) {
        if (arr[index] % 2 === 0) {
            sumEven += Number(arr[index]);
        } else sumOdd += Number(arr[index]);
    }

    return output += `Odd sum = ${sumOdd}, Even sum = ${sumEven}`
}

console.log(oddAndEvenSum(3495892137259234));